faunadb/streams/client.py

```diff
@@ -14,7 +14,7 @@
 from .errors import StreamError
 from .events import Error, parse_stream_request_result_or_none
 
-VALID_FIELDS = {"diff", "prev", "document", "action"}
+VALID_FIELDS = {"diff", "prev", "document", "action", "set"}
```

Comment:
Isn't the added field `index`? I don't see a test for the stream option fields. When I try to use `"index"`, I get:

```
Exception: Valid fields options are {'action', 'prev', 'document', 'diff', 'set'}, provided ['action', 'document', 'index', 'prev', 'diff'].
```

Reply:
You're right. That was a typo. Thank you.